src/Bicep.LangServer/IServiceCollectionExtensions.cs

```diff
@@ -24,7 +24,7 @@ public static class IServiceCollectionExtensions
     public static IServiceCollection AddBicepCore(this IServiceCollection services) => services
         .AddSingleton<INamespaceProvider, DefaultNamespaceProvider>()
         .AddSingleton<IResourceTypeLoader, AzResourceTypeLoader>()
```

Comment:
Does this need to be removed, too?

Reply:
yes, overlooked its removal. Removing now